src/components/PostPageDetail/index.js

```diff
@@ -1,39 +1,39 @@
-import {Dimensions, Keyboard, ScrollView, StatusBar, StyleSheet, View} from 'react-native';
 import {useNavigation} from '@react-navigation/core';
 import {useRoute} from '@react-navigation/native';
+import moment from 'moment';
+import * as React from 'react';
+import {Dimensions, Keyboard, ScrollView, StatusBar, StyleSheet, View} from 'react-native';
+import Toast from 'react-native-simple-toast';
 
-import BlockComponent from '../BlockComponent';
-import ContainerComment from '../Comments/ContainerComment';
-import Content from './elements/Content';
+import {Footer} from '..';
+import {Context} from '../../context';
+import {saveComment} from '../../context/actions/comment';
+import {setFeedByIndex} from '../../context/actions/feeds';
+import {useFeedDataContext} from '../../hooks/useFeedDataContext';
+import usePostContextHook, {CONTEXT_SOURCE} from '../../hooks/usePostContextHooks';
 import Header from '../../screens/FeedScreen/Header';
-import LoadingWithoutModal from '../LoadingWithoutModal';
-import ShareUtils from '../../utils/share';
-import StringConstant from '../../utils/string/StringConstant';
-import WriteComment from '../Comments/WriteComment';
 import useFeed from '../../screens/FeedScreen/hooks/useFeed';
-import usePostDetail from './hooks/usePostDetail';
-import useWriteComment from '../Comments/hooks/useWriteComment';
-import usePostContextHook, {CONTEXT_SOURCE} from '../../hooks/usePostContextHooks';
+import {createCommentParentV3, getCommentList} from '../../service/comment';
+import {getFeedDetail} from '../../service/post';
+import {downVote, upVote} from '../../service/vote';
+import {showScoreAlertDialog} from '../../utils/Utils';
 import {
   ANALYTICS_SHARE_POST_FEED_ID,
   ANALYTICS_SHARE_POST_PDP_SCREEN,
   SOURCE_PDP
 } from '../../utils/constants';
-import {Context} from '../../context';
-import {Footer} from '..';
-import {createCommentParentV3, getCommentList} from '../../service/comment';
-import {downVote, upVote} from '../../service/vote';
 import {fonts} from '../../utils/fonts';
 import {getCountCommentWithChildInDetailPage} from '../../utils/getstream';
-import {getFeedDetail} from '../../service/post';
-import {saveComment} from '../../context/actions/comment';
-import {setFeedByIndex} from '../../context/actions/feeds';
-import {showScoreAlertDialog} from '../../utils/Utils';
-import {useFeedDataContext} from '../../hooks/useFeedDataContext';
+import ShareUtils from '../../utils/share';
+import StringConstant from '../../utils/string/StringConstant';
+import BlockComponent from '../BlockComponent';
+import ContainerComment from '../Comments/ContainerComment';
+import WriteComment from '../Comments/WriteComment';
+import useWriteComment from '../Comments/hooks/useWriteComment';
+import LoadingWithoutModal from '../LoadingWithoutModal';
 import {withInteractionsManaged} from '../WithInteractionManaged';
+import Content from './elements/Content';
+import usePostDetail from './hooks/usePostDetail';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been reordered and grouped by their source directory. This is a good practice for maintainability and readability, as it makes it easier to find and manage imports. However, there are no functional changes in this hunk.

```diff
+  import {useNavigation} from '@react-navigation/core';
+  import {useRoute} from '@react-navigation/native';
+  import moment from 'moment';
+  import * as React from 'react';
+  import {Dimensions, Keyboard, ScrollView, StatusBar, StyleSheet, View} from 'react-native';
+  import Toast from 'react-native-simple-toast';
+  
+  import {Footer} from '..';
+  import {Context} from '../../context';
+  import {saveComment} from '../../context/actions/comment';
+  import {setFeedByIndex} from '../../context/actions/feeds';
+  import {useFeedDataContext} from '../../hooks/useFeedDataContext';
+  import usePostContextHook, {CONTEXT_SOURCE} from '../../hooks/usePostContextHooks';
+  import Header from '../../screens/FeedScreen/Header';
+  import useFeed from '../../screens/FeedScreen/hooks/useFeed';
+  import {createCommentParentV3, getCommentList} from '../../service/comment';
+  import {getFeedDetail} from '../../service/post';
+  import {downVote, upVote} from '../../service/vote';
+  import {showScoreAlertDialog} from '../../utils/Utils';
+  import {
+    ANALYTICS_SHARE_POST_FEED_ID,
+    ANALYTICS_SHARE_POST_PDP_SCREEN,
+    SOURCE_PDP
+  } from '../../utils/constants';
+  import {fonts} from '../../utils/fonts';
+  import {getCountCommentWithChildInDetailPage} from '../../utils/getstream';
+  import ShareUtils from '../../utils/share';
+  import StringConstant from '../../utils/string/StringConstant';
+  import BlockComponent from '../BlockComponent';
+  import ContainerComment from '../Comments/ContainerComment';
+  import WriteComment from '../Comments/WriteComment';
+  import useWriteComment from '../Comments/hooks/useWriteComment';
+  import LoadingWithoutModal from '../LoadingWithoutModal';
+  import {withInteractionsManaged} from '../WithInteractionManaged';
+  import Content from './elements/Content';
+  import usePostDetail from './hooks/usePostDetail';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->